store/sqlstore/terms_of_service_store.go

```diff
@@ -55,8 +56,18 @@ func (s SqlTermsOfServiceStore) Save(termsOfService *model.TermsOfService) (*mod
 func (s SqlTermsOfServiceStore) GetLatest(allowFromCache bool) (*model.TermsOfService, error) {
 	var termsOfService *model.TermsOfService
 
-	err := s.GetReplica().SelectOne(&termsOfService, "SELECT * FROM TermsOfService ORDER BY CreateAt DESC LIMIT 1")
+	query := s.getQueryBuilder().
+		Select("*").
+		From("TermsOfService").
+		OrderBy("CreateAt DESC").
+		Limit(uint64(1))
+
+	queryString, args, err := query.ToSql()
 	if err != nil {
+		return nil, model.NewAppError("SqlTermsOfServiceStore.GetLatest", "store.sql.build_query.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
We are no longer using `AppError` in the store, please use regular errors.

Reply:
Hi @jespino ,

Can I confirm that this would be in the generic std method of using `errors.New`?

Cheers.

Reply:
`errors.Wrap` for sure, something like `errors.Wrap(err, "termofservice_getlatest_tosql")` or something like that should work.